intermediate-javascript-calorie-counter/index180.js

```diff
@@ -0,0 +1,38 @@
+document.getElementById('calorie-form').onsubmit = calculate;
+
+function calculate(e) {
+  e.preventDefault();
+
+  const total = Array.from(document.getElementsByClassName('cal-control'))
+    .map(input => Number(input.value))
+    .reduce((accumulator, currentValue) => {
+      /*code to run*/
+    }, 0);
+}
+/*
+I DON'T KNOW IF THIS MAKES SENSE, COULD USE A SECOND PAIR OF EYES
```

Comment:
Your explanation is very good, but it will have to be shortened. Perhaps showing the code for reduce on the array `[1, 3, 5]`, then briefly explaining what it's doing on each iteration before telling the student what to write in reduce would be best.